common/lib/xmodule/xmodule/tabs.py

```diff
@@ -570,14 +567,46 @@ def check(actual_dict, raise_error=True):
     return check
 
 
-def link_reverse_func(reverse_name):
+def course_reverse_func(reverse_name):
+    """
+    Returns a function that will determine a course URL for the provided
+    reverse_name.
+
+    See documentation for course_reverse_func_from_name_func.  This function
+    simply calls course_reverse_func_from_name_func after wrapping reverse_name
+    in a function.
     """
-    Returns a function that takes in a course and reverse_url_func,
-    and calls the reverse_url_func with the given reverse_name and course's ID.
+    return course_reverse_func_from_name_func(lambda course: reverse_name)
 
-    This is used to generate the url for a CourseTab without having access to Django's reverse function.
+
+def course_reverse_func_from_name_func(reverse_name_func):
```

Comment:
@andy-armstrong As someone else who cares about good and concise names, do you have a proposal for a different name here?  I don't and neither does @dianakhuang, although neither of us love the current name.  With no new proposal, I'll just leave this as is.